Tools/Scripts/webkitpy/w3c/test_parser.py

```diff
@@ -122,8 +122,6 @@ def analyze_test(self, test_contents=None, ref_contents=None):
             test_info = {'test': self.filename, 'crashtest': True}
         elif test_type == "testharness":
             test_info = {'test': self.filename, 'jstest': True}
-        elif test_type == "support" and wpt_sourcefile.name_is_reference:
-            test_info = {'referencefile': self.filename}
```

Comment:
Isn't this metadata useful to have around? Would it be better to delete this instead: https://searchfox.org/wubkat/rev/088a3e573b59664dc988291e52900bba6b106b2c/Tools/Scripts/webkitpy/w3c/test_importer.py#387-389 ?

Reply:
Why would we need the metadata? It's… meaningless. Though it does make me realise that maybe we should just totally remove `referencefile`; there's nothing meaningfully different with them versus arbitrary support files.

Reply:
OK, now actually totally removed the metadata (and updated the commit message to reflect that).